java/core/src/main/java/sleeper/core/schema/Schema.java

```diff
@@ -150,9 +149,9 @@ public int hashCode() {
     }
 
     public static final class Builder {
-        private List<Field> rowKeyFields;
-        private List<Field> sortKeyFields;
-        private List<Field> valueFields;
+        private List<Field> rowKeyFields = new ArrayList<>();
+        private List<Field> sortKeyFields = new ArrayList<>();
+        private List<Field> valueFields = new ArrayList<>();
```

Comment:
This initialisation is not necessary with the validateXKeys methods. It might be a little misleading.